wiki/Game_mode/osu!taiko/en.md

```diff
@@ -24,24 +24,22 @@ Alternatively, click on the `Mode` button and select `osu!taiko`.
 
 For players with prior *Taiko no Tatsujin* experience:
 
-- No chibi dancers at the bottom (must be storyboarded)
-- The health bar must be at least 50% full to pass the song (which is not really obvious using the default skin; please look in the [Skin Compendium](https://osu.ppy.sh/community/forums/topics/180864) for a decent osu!taiko skin)
+- No PuchiChara at the bottom (must be storyboarded)
+- The health bar must be at least 50% full to pass the song
 - *Kiai Time* triggers the *"Go-Go Time"*
   - Gameplay gimmicks like the hit balloons or forked paths are not implemented (only the barebone basics)
-- Background display difference
-  - osu!taiko-specific beatmap usually has its own custom background, leaving upper section empty
-  - A video or static background image-only beatmap will take the lower portion only
-  - If there is an element of storyboard, it will take over all the available screen space except the gameplay-important parts
-
-For newcomers of osu!taiko mode, the screen is split to two different sections; the upper section contains the gameplay elements while the lower section contains an image or video for the beatmap. At the upper section, there is the health bar, which unlike other game modes, starts off empty and must be filled up to at least 50% full or half-full to pass the difficulty. At the top-middle part below the health bar, a taiko drum is located at the left-side and a moving conveyor belt which carries the hit objects from right to left passing through a double white circles which serves as judgement circle near the taiko drum. Above the taiko drum is the animated taiko mascot which will react based on judgement received along with upper background colour changes. Lastly, there is the usual score, accuracy, and timer of the beatmap at the upper-right.
+- Background difference
+  - A video or static background image-only beatmap will take up the lower portion of the screen
+  - If the map has been storyboarded, the storyboard will take over the bottom of the screen
+For newcomers of osu!taiko mode, the screen is split into two different sections; the upper section contains the gameplay elements while the lower section contains an image, video, or storyboard for the beatmap. In the upper section, there is the health bar, which unlike other game modes, starts empty and must be filled up to at least 50% full or half-full to pass the difficulty. At the top-middle part below the health bar, a taiko drum is located at the left side and a moving conveyor belt which carries the hit objects from right to left passing through the judgement circle near the taiko drum. Above the taiko drum is an animated taiko mascot (otherwise known as a pippidon or don) which will react based on kiai sections, combo milestones, and notes missed during kiai time along with a scrolling background slider that changes colour when notes are missed. Lastly, there is the usual score, accuracy, and timer of the beatmap at the upper-right.
```

Comment:
while I can see some things changed for better, it's still a stream of text readable poorly enough to make me either backtrace or double the efforts required to following. not your fault, but that gives your change little to no benefit

Reply:
(if anything this doesn't require a reply -- more of a side comment I made without realising neither action would help the article in its current shape anyway)

Reply:
made it quite a bit shorter and easier to read